connectors/sources/onedrive.py

```diff
@@ -303,12 +301,19 @@ async def paginated_api_call(self, url_name, **url_kwargs):
                 break
 
     async def list_users(self):
-        async for response in self.paginated_api_call(url_name=USERS):
+        async for response in self.paginated_api_call(
+            url_name=USERS, page_size=FETCH_SIZE
```

Comment:
I think it's better to make `FETCH_SIZE` as the default value of `page_size`, so you can call it with `self.paginated_api_call(url_names=USERS)